### PR TITLE
Remove obsolete coding conventions; Add/update some relevant ones

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Subj.  I did not try to add anything new, just documenting some of our current agreements

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #3860** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3860?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3860?utm_source=stack-comment).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,36 +53,19 @@ Exceptions:
...
-- `copy`: Copy update
-- `docs`: Documentation only changes
-- `feat`: A new feature
-- `fix`: Bug fixes
-- `refactor`: A code change that neither fixes a bug nor adds a feature
-- `revert`: Reverts a previous commit/PR
-- `style`: Changes that do not affect the meaning of the code (white-space, formatting etc)
-- `test`: Adding tests or correcting existing tests
+No requirements, go nuts if you want :)
+We had previously enforced naming conventions for branches, and it had not shown enough value to be worth it
```
Comment (gustaveen) [inline CONTRIBUTING.md:62]: nit: maybe refer to the PR again, usually more useful to put that info there

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -53,36 +53,19 @@ Exceptions:
...
 
 ## Pull Requests and Code Reviews
 
+We mostly use Graphite for Pull Requests as it provides better experience and developer productivity compared to Github PRs.
```
Comment (gustaveen) [inline CONTRIBUTING.md:66]: nit: maybe we can highlight that it's primarly the stacking feature that gives us the most value with small PRs and easy merging

**Comment by alebedev** [comment]
### Merge activity

* **Jan 22, 10:26 AM**: @alebedev merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3860).

**Pull request merged**